src/app/features/collection/components/chip-list/chip-list.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="chip-list">
+  @for (chip of chipList(); track chip) {
+    <app-chip
+      [chipId]="chip.id"
```

Comment:
i think we can agreed on signature with object as input, to avoid multiple inputs, since we will receive objects from API and probably just pass them to collection page as it is